moto/ecs/models.py

```diff
@@ -652,18 +652,21 @@ def stop_task(self, cluster_str, task_str, reason):
         raise Exception("Could not find task {} on cluster {}".format(
             task_str, cluster_name))
 
-    def create_service(self, cluster_str, service_name, task_definition_str, desired_count):
+    def create_service(self, cluster_str, service_name, task_definition_str, desired_count, load_balancers=None):
         cluster_name = cluster_str.split('/')[-1]
         if cluster_name in self.clusters:
             cluster = self.clusters[cluster_name]
         else:
             raise Exception("{0} is not a cluster".format(cluster_name))
         task_definition = self.describe_task_definition(task_definition_str)
         desired_count = desired_count if desired_count is not None else 0
+        load_balancers = load_balancers if load_balancers is not None else []
+
         service = Service(cluster, service_name,
                           task_definition, desired_count)
```

Comment:
Would it not make more sense to add load_balancers=None to the constructor of `Service` and then if provided extend `self.load_balancers`?

Reply:
Initially, my solution was as you said. Agree that makes more sense, changing back